client/app/components/packages/scope-of-work-draft/edit.hbs

```diff
@@ -0,0 +1,97 @@
+<SaveableForm 
+  @model={{@package}} 
+  @validators={{array (hash) (hash)}} 
+  as |saveableForm|
+>
+
+  <div class="grid-x grid-margin-x">
+    <div class="cell large-8">
+
+      <section class="form-section">
+        <h1 class="header-large">
+          Draft Scope of Work (SOW)
+          <small class="text-weight-normal"
+            data-test-package-dcpPackageversion
+          >
+            {{if @package.dcpPackageversion (concat '(V' @package.dcpPackageversion ')')}}
+          </small>
+        </h1>
+
+        <h2 class="no-margin"
+          data-test-project-dcpProjectname
+        >
+          {{@package.project.dcpProjectname}}
+          <small class="text-weight-normal"
+            data-test-project-dcpName
+          >
+            {{if @package.project.dcpName (concat '(' @package.project.dcpName ')')}}
+          </small>
+        </h2>
+
+        <p class="text-large text-dark-gray"
+          data-test-project-dcpBorough
+          data-test-package-statuscode
+        >
+          {{optionset 'bbl' 'boroughs' 'label' @package.project.dcpBorough}} |
+          {{optionset 'package' 'statuscode' 'label' @package.statuscode}}
+        </p>
+      </section>
+      
+      {{#if @package.dcpPackagenotes}}          
+        <saveableForm.Section @title="Package Notes from City Planning Staff"
+          data-test-package-notes
+        >
+          {{@package.dcpPackagenotes}}
+        </saveableForm.Section>
+      {{/if}}
+
+      <Packages::ScopeOfWorkDraft::AttachedDocuments
+        @form={{saveableForm}}
+        @model={{@package}}
+      />
+    </div>
+    
+    <div class="cell large-4 sticky-sidebar">
+      <saveableForm.PageNav>
+        <saveableForm.SaveButton 
+        @isEnabled={{or @package.isDirty saveableForm.isSaveable}}
+        @onClick={{this.savePackage}} 
+        data-test-save-button 
+      />
+        
+        <saveableForm.SubmitButton
+          @isEnabled={{saveableForm.isSubmittable}}
+          class="secondary"
+          data-test-submit-button
+        />
+      </saveableForm.PageNav>
+
+      <saveableForm.ConfirmationModal
+        @action={{component saveableForm.SubmitButton
+          onClick=this.submitPackage
+          isEnabled=saveableForm.isSubmittable
+          class="no-margin"
+        }}
+        @footer={{component 'packages/scope-of-work-draft/scope-of-work-draft-error'
```

Comment:
The folder naming needs to be consistent. Please change this to `draft-scope-of-work/draft-scope-of-work-error`

Reply:
Let's check with @allthesignals first!

Reply:
@godfreyyeung yep, I know we had a conversation, but @CeciliaCY asked about this in chat and I suggested scope-of-work-draft. I still stand by that, as you know! TYVM all!